Resources/Textures/DeltaV/Objects/Specific/Hydroponics/Cosmic_Revenant.rsi/meta.json

```diff
@@ -0,0 +1,87 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13, editted by Alchebun",
```

Comment:
```suggestion
  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13, edited by Alchebun",
```
:trollface:

Reply:
fixed. I know you know that while spelling matters a lot in coding, this isn't one of those times. update named accordingly.